### PR TITLE
Batch Changes Q2 strat update 

### PR DESCRIPTION
**Pull request by malomarrec**

### CODE REVIEW

File: content/strategy-goals/strategy/code-graph/batch-changes/index.md [hunk 1]
```diff
@@ -13,33 +13,33 @@ Batch Changes is a tool to find code that needs to be changed and change it at s
 - [Planning board](https://github.com/orgs/sourcegraph/projects/216)
 - [Roadmap](https://github.com/orgs/sourcegraph/projects/214/views/34?filterQuery=owning-org%3A%22Code+Graph%22+type%3ARoadmap+owning-team%3A%22Batch+changes%22)
 
-## Vision
+## Definitions
```
Comment (jasonyavorsky) [inline content/strategy-goals/strategy/code-graph/batch-changes/index.md:16]: Nice addition!

File: content/strategy-goals/strategy/code-graph/batch-changes/index.md [hunk 2]
```diff
@@ -73,13 +73,14 @@ The key [success metric](../../../../departments/product-engineering/engineering
 
 ### Time-to-value
 
-Batch Changes is a powerful product with a steep learning curve. The main friction point is that Batch Changes requires users to write **steps** (the components of a batch spec that define a specific change to be executed), either using existing code rewrite tools (sed, comby, etc) or writing code from scratch. Most developers are unfamiliar with such tooling, and have a hard time onboarding to batch changes.
+Batch Changes is a powerful product with a steep learning curve. The main friction point is that Batch Changes requires users to write **code rewrite steps** (the components of a batch spec that define a specific change to be executed), either using existing code rewrite tools (sed, comby, etc) or writing code from scratch. Most developers are unfamiliar with such tooling, and have a hard time onboarding to Batch Changes.
 
-We want to decrease time to value and make it easier for any developer to get value out of Batch Changes. We measure success here by tracking:
+We want to decrease time to value and make it easier for any developer to get value out of Batch Changes for a use case. We will measure success here by tracking:
```
Comment (jasonyavorsky) [inline content/strategy-goals/strategy/code-graph/batch-changes/index.md]: Maybe linking to the use cases would be nice here.

**Review by jasonyavorsky — APPROVED** [review]
Great update! Please leave it open for a few days for cross-team feedback/questions.

**Review by eseliger — APPROVED** [review]
🌟 lets gooo

**Review by courier-new — APPROVED** [review]
Great updates, thanks Malo!!

**Review by LawnGnome — APPROVED** [review]

**Pull request merged**